src/main/java/com/example/bookstore/service/UserService.java

```diff
@@ -0,0 +1,15 @@
+package com.example.bookstore.service;
+
+import com.example.bookstore.dto.UserRegistrationRequestDto;
+import com.example.bookstore.dto.UserResponseDto;
+import com.example.bookstore.exception.RegistrationException;
+import com.example.bookstore.model.User;
+import jakarta.validation.Valid;
+import org.springframework.web.bind.annotation.RequestBody;
+
+public interface UserService {
+    UserResponseDto register(@RequestBody @Valid UserRegistrationRequestDto request)
```

Comment:
```suggestion
    UserResponseDto register(UserRegistrationRequestDto request)
```
these annotations are redundant here